incubator/spark-history-server/values.yaml

```diff
@@ -0,0 +1,54 @@
+replicaCount: 1
+
+namespace: spark-history-server
+createNamespace: true
+clusterRoleName: spark-hs
+clusterRoleBindingName: spark-hs
+
+image:
+  repository: lightbend/spark
+  tag: k8s-daily
+  pullPolicy: Always
+
+service:
+  type: LoadBalancer
+  port: 18080
```

Comment:
Is the `1` at the start of `18080` a typo?

Reply:
No. The Spark history server by default exposes the UI on port 18080, as documented [here](https://spark.apache.org/docs/latest/monitoring.html).